util/dvsim/dvsim.py

```diff
@@ -601,9 +601,13 @@ def main():
 
     args.scratch_root = resolve_scratch_root(args.scratch_root)
     args.branch = resolve_branch(args.branch)
-    args.cfg = os.path.abspath(args.cfg)
     proj_root = resolve_proj_root(args)
     log.info("[proj_root]: %s", proj_root)
+    if args.remote:
+        cfg_path = os.getcwd().replace(get_proj_root() + "/", "")
+        args.cfg = os.path.join(proj_root, cfg_path, args.cfg)
+    else:
+        args.cfg = os.path.abspath(args.cfg)
```

Comment:
Something like this is probably a bit more clear. 
```suggestion
    # Have resolve_proj_root return both, the src and dest
    proj_root_src, proj_root_dest = resolve_proj_root(args)

    args.cfg = os.path.abspath(args.cfg)
    if args.remote:
        cfg_path = args.cfg.replace(proj_root_src + "/", "")
        args.cfg = os.path.join(proj_root_dest, cfg_path)
```

Reply:
Thanks Sri for the suggestion. Could you take another look and see if the updated change meets your expectation?